kickstart/provision.erb

```diff
@@ -143,6 +144,11 @@ echo "updating system time"
 # update all the base packages from the updates repository
 yum -t -y -e 0 update
 
+<% if chef_enabled %>
+echo "Bootstraping chef"
+<%= respond_to?(:chef_bootstrap) ? chef_bootstrap(@host) : snippet_if_exists("chef-client omnibus bootstrap") %>
```

Comment:
I see... As we're having the salt snippets also in community-templates, could you add the chef ones also here, please?